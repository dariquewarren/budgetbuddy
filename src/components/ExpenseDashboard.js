import React from 'react'
import ExpenseList from './ExpenseList'
import ExpenseListFilters from './ExpenseListFilters'

const ExpenseDashboard = ()=>(
    <div>
    this is from my dash component
    <ExpenseListFilters/>
    <ExpenseList/>

    </div>
    )

export default ExpenseDashboard







// import React from 'react'
// import {BrowserRouter, Route, Switch, Link, NavLink} from 'react-router-dom'




// export default 