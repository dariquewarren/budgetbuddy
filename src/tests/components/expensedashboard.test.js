import React from 'react'
import {shallow} from 'enzyme'
import ExpenseDashboard from '../../components/ExpenseDashboard'

test('dashboard component', ()=>{
    const wrapper = shallow(<ExpenseDashboard/>)
    expect(wrapper).toMatchSnapshot()
})