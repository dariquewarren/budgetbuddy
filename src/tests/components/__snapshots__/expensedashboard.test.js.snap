// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dashboard 1`] = `
<div>
  <Connect(ExpensesSummary) />
  this is from my dash component
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
